magical_change_part1/lib/widgets/custom_card.dart

```diff
@@ -0,0 +1,89 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import 'package:magical_change_part1/users_model.dart';
+
+class CustomCard extends StatelessWidget {
+  UserModels user;
+  VoidCallback ontap;
+  CustomCard({super.key, required this.user, required this.ontap});
+
+  @override
+  Widget build(BuildContext context) {
+    return Card(
+      child: Padding(
+        padding: EdgeInsets.all(10),
+        child: Row(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            CircleAvatar(
+              backgroundImage: AssetImage(user.avatar),
+              maxRadius: 25,
+            ),
+            SizedBox(width: 20),
+            Padding(
+              padding: EdgeInsets.all(8.0),
+              child: Column(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                mainAxisAlignment: MainAxisAlignment.start,
+                children: [
+                  Row(
+                    children: [
+                      SizedBox(
+                        width: 120,
+                        child: Flexible(
+                          child: Text(
+                            "${user.name}",
+                            style: TextStyle(
+                              fontSize: 14,
+                              fontWeight: FontWeight.w700,
+                            ),
+                          ),
+                        ),
+                      ),
+                      SizedBox(width: 80),
+                      Text(
+                        "${user.phoneNumber}",
```

Comment:
here "${}" is not required as you need to display value of a single parameter. Use this when you need string interpolation.